custom_dc/README.md

```diff
@@ -32,6 +32,58 @@ Additionally, a custom DC combines its own local datasets with base DC datasets
   You can use a sample key `AIzaSyA9RsPS8tBCKwrET3qqAzydhOMWP0Ee8Y8` for local
   development as discussed below.
 
+## Quick Start
```

Comment:
Wonder should quick start be in a separate file and refer to that file from here?

Reply:
Okay, then should call out this is a quick start and should read the rest of the sections for full guides.

Reply:
Done.